src/components/molecules/Visualizer/Engine/ref.ts

```diff
@@ -1,3 +1,4 @@
+import Rectangle from "cesium/Source/Core/Rectangle";
```

Comment:
```suggestion
```

Please do not use objects of Cesium directly because Re:Earth Engine API should be independent from a specific map engine.

```ts
import { Rect } from "@reearth/util/value";
```